web/source/dom/domEventHandlers.ts

```diff
@@ -352,7 +352,7 @@ namespace com.keyman.dom {
         if(el && typeof el.kmwInput != 'undefined' && el.kmwInput == false) {
           return true;
         }
-      } else if(el && el.className.indexOf('kmw-disabled') >= 0) {
+      } else if(el?.className?.indexOf('kmw-disabled') >= 0) {
```

Comment:
```suggestion
      } else if(el?.classList?.indexOf('kmw-disabled') >= 0) {
```

Tidier because it's not a substring match and `classList` is [supported](https://caniuse.com/?search=classlist) in all our environments (`includes()` is not, although probably okay because of es6-shim?)

Reply:
Would that also apply to domManager? It's got several uses (I only highlighted the first two)

https://github.com/keymanapp/keyman/blob/d1ab61eac11f9601931752d56336af707b97c2e0/web/source/dom/domManager.ts#L818-L826

Maybe a future PR?

Reply:
> Maybe a future PR?

OK